extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -0,0 +1,94 @@
+<div class="wds-global-navigation">
+	<div class="wds-global-navigation__content-bar">
+		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-logo">
+			<?= DesignSystemHelper::getSvg(
+				'wds-company-logo-fandom',
+				'wds-global-navigation__content-item-logo-fandom'
+			) ?>
+			<span class="wds-global-navigation__content-item-logo-powered-by">powered by</span>
+			<?= DesignSystemHelper::getSvg(
+				'wds-company-logo-wikia',
+				'wds-global-navigation__content-item-logo-wikia'
+			) ?>
+		</a>
+		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-is-games">Games</a>
+		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-is-movies">Movies</a>
+		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-is-tv">TV</a>
+		<div class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-dropdown">
+			<div class="wds-dropdown-toggle wds-global-navigation__dropdown-toggle">
+				<span>Wikis</span>
+				<?= DesignSystemHelper::getSvg(
+					'wds-icons-dropdown-tiny',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wds-icons-dropdown-tiny" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)